### ISSUE TITLE
Methods to rank feature importance

### ISSUE THREAD

**Issue by sls-mdr** [jb-ds2020/nf-ds3-capstone-churn-prevention#12]

**Comment by sls-mdr** [comment]
**Here are a few for a closer look:**
from sklearn.feature_selection import SelectKBest
from sklearn.feature_selection import chi2
from sklearn.feature_selection import RFE
from sklearn.feature_selection import SelectFromModel

**Plot:** 
df.drop('churn', axis=1).corrwith(df.churn).sort_values().plot(kind='barh', figsize=(10, 30))

**Implementation example:**
https://github.com/Ela-Bo/HR-Analytics-Employee-Attrition/blob/main/employee%20attrition.ipynb
